contents/product-engineers/dogfooding.md

```diff
@@ -0,0 +1,137 @@
+
+To do this yourself is a two step process:
+
+### 1. Create a culture of dogfooding
+
+A hot take: forcing your team to dogfood your product is a bad sign. This means there is a fundamental mismatch between the work your team wants to do and the product you are building. There are a few potential reasons for this:
+
+1. Your team isn't the ideal user for your product.
+2. Your product isn't good enough for your team to use.
+3. You haven't created the right culture to support dogfooding.
+
+The first two are major problems beyond the focus of this article (they require a rethinking of your entire strategy). The third is the most common and fixable.
+
+To create a culture of dogfooding, first, you need buy-in from the top. The responsibility to dogfood does not belong to one person or team. The entire organization needs to be bought into using the product, providing feedback, and continuously improving the product. 
+
+Everyone at PostHog, from the cofounders to the marketing team, uses PostHog and provides feedback on their usage.  
+
+![Tim dogfooding](https://res.cloudinary.com/dmukukwp6/image/upload/cofounder_ec648e21d3.png)
+
+For feedback is critical to the dogfooding process and to make people comfortable with giving feedback, you need trust.  Your team must trust each other to give and [receive feedback](/handbook/people/feedback). Assume positive intent, acknowledge it, and do with it what you want. This might be uncomfortable, but dogfooding doesn't work without it.
+
+### 2. Implement dogfooding into product development
+
+After creating the culture to support dogfooding, it must be integrated into your process for improving your product. When done right, it can have an impact on multiple stages from ideation to testing to post-release. 
+
+We've found three details are key to getting the most out of dogfooding:
+
+1. **Treat your team like a customer.** [Interview them](/newsletter/talk-to-users), ask them for feedback on what you built, use this to guide your next iteration, and repeat. For the best results, go out of your way to ask for feedback and make your ask specific.
+
+2. **Create tighter feedback loops.** At PostHog, we have [product engineers](/blog/what-is-a-product-engineer) who take full ownership of a product. They both figure out what to build and build it. This means they dogfood and can implement the improvements from dogfooding, iterating rapidly. If you never action feedback, your dogfooding is going to waste.
+
+3. **Separate deployment from release.** Doing this enables you to dogfood a change without releasing it to all your users. Using [feature flags](/feature-flags), you can [test in production](/product-engineers/testing-in-production), fix issues, and ship a polished product to users. 
+
+## The benefits of dogfooding
```

Comment:
feels strange to have the benefits right at the end, i'd move this section up so it comes before 'how to dogfood well' and then we can end on the pitfalls